client/src/main/scala/org/http4s/client/package.scala

```diff
@@ -1,6 +1,15 @@
 package org.http4s
 
-package object client extends ClientTypes
+import scala.concurrent.duration.DurationLong
+
+package object client extends ClientTypes {
+
+  object defaults {
```

Comment:
I would expect this to generate a compiler warning about not putting objects in a package object.  I think this wants to live at the top-level in the `client` package.

Reply:
Although now I see we've already done this in `server`. :thinking: